csaf_2.1/prose/edit/src/tests-02-opt-41-old-epss-timestamp.md

```diff
@@ -0,0 +1,58 @@
+### Old EPSS Timestamp
```

Comment:
I can add the corresponding section label entries after merge into the revision branch.

Reply:
Thank you.

Reply:
I just added them in [3b57346](https://github.com/oasis-tcs/csaf/pull/969/commits/3b57346a5c4bd48a511058de0c3b9cd2371a04e7) as well as put that into the bind.txt.